synfig-studio/src/gui/app.cpp

```diff
@@ -1754,18 +1735,33 @@ App::App(const synfig::String& basepath, int *argc, char ***argv):
 		get_ui_interface()->error(_("Unknown exception caught when constructing App.\nThis software may be unstable."));
 	}
 
-	// Load sound effects
-	sound_render_done = new SoundProcessor();
-	sound_render_done->addSound(
-		SoundProcessor::PlayOptions(),
-		SoundProcessor::Sound(ResourceHelper::get_sound_path("renderdone.wav")));
+	// FIXME(rodolforg): Gtk::Application quits when all windows are hidden/closed
+	// DockManager::read_widget(std::string &x) hides MainWindow and
```

Comment:
DockManager hides MainWindow to fix this issue (https://github.com/synfig/synfig/issues/1975).

I need a time to check, if this issue is reproduced with Gtk::Application (maybe it works differently).